public/main.css

```diff
@@ -0,0 +1,41 @@
+/* line 2, ../src/stylesheets/globals/_reset.scss */
+* {
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -o-box-sizing: border-box;
+}
+
+/* line 9, ../src/stylesheets/globals/_reset.scss */
+html {
+  font-size: 62.5%;
+}
+
+/* line 1, ../src/stylesheets/components/_inventory.scss */
+.inventory--MAIN {
+  background-color: #eee;
+  overflow: auto;
+}
+/* line 5, ../src/stylesheets/components/_inventory.scss */
+.inventory--MAIN .inputfield--MAIN {
```

Comment:
No, it's just a BEM-like syntax I use for CSS, personal preference but it's the best way I've found for organizing CSS without losing your mind.